### PR TITLE
[8.0]purchase_rfq_number

### PR DESCRIPTION
**Pull request by mikevhe18**
Adds a sequence on purchase RFQ

### CODE REVIEW

File: purchase_rfq_number/README.rst [hunk 1]
```diff
@@ -0,0 +1,74 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+    
+==============================================
+Purchase RFQ(Request For Quotation) Numeration
```
Comment (elicoidal) [inline purchase_rfq_number/README.rst]: Purchase RFQ (Request For Quotation) Numbering

File: purchase_rfq_number/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# © 2016 OpenSynergy Indonesia
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Purchase RFQ(Request For Quotation) Numeration',
```
Comment (elicoidal) [inline purchase_rfq_number/__openerp__.py]: Purchase RFQ(Request For Quotation) Numeration

File: purchase_rfq_number/README.rst [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+==============================================
+
+This module adds the possibility to define a sequence for the purchase RFQ.
+When you create a RFQ, RFQ Number will be automatically filled in accordance with the sequence.
```
Comment (elicoidal) [inline purchase_rfq_number/README.rst]: s/accordance with/according to

**Comment by mikevhe18** [comment]
@elicoidal thank you for the review.

**Comment by andhit-r** [comment]
@mikevhe18 no unittest?

**Comment by andhit-r** [comment]
Functionality tested on runbot :+1:

**Comment by mikevhe18** [comment]
hi @gurneyalex , would you kindly review this module ?

**Comment by mikevhe18** [comment]
Hi, @jgrandguillaume and @yvaucher .
This PR has been more than 5 days.
Can you help me to reviewing this PR again and give a vote ?

Thanks in advance.

**Comment by andhit-r** [comment]
@OCA/procurement-purchase-maintainers would you kindly give this PR needs reviews label. Many thanks

**Comment by yvaucher** [comment]
LGTM :+1:

**Comment by azmimr67** [comment]
Runbot Tested :+1:

**Comment by andhit-r** [comment]
@OCA/procurement-purchase-maintainers can we merged this? Thanks

**Pull request merged**